test/unit/Fixture/MethodsOrder.php

```diff
@@ -9,7 +9,7 @@ public function forth();
 
 class MethodsOrderParent
 {
-    public function third()
+    public function second()
```

Comment:
We should probably have a new fixture here, instead of changing the existing one (risky)

Reply:
There's new fixture but this test is broken in master so this is fix - https://3v4l.org/n5MhC

Reply:
That didn't parse for me, sorry: you mean this test didn't run on `master`? Or is the change introducing a breakage?

Reply:
The expectation in master is wrong.